src/interface/src/app/treatments/direct-impacts-map/direct-impacts-map.component.ts

```diff
@@ -38,23 +41,10 @@ export class DirectImpactsMapComponent {
     private authService: AuthService
   ) {}
 
-  readonly labels: Record<Year, string> = {
-    cero: 'Baseline',
-    five: '5 Years',
-    ten: '10 Years',
-    fifteen: '15 Years',
-    twenty: '20 Years',
-  };
-
-  readonly variables: Record<Year, string> = {
-    cero: 'delta_0',
-    five: 'delta_5',
-    ten: 'delta_10',
-    fifteen: 'delta_15',
-    twenty: 'delta_20',
-  };
+  readonly labels = YEAR_INTERVAL_LABELS;
 
-  @Input() year: Year = 'cero';
+  readonly variables = YEAR_INTERVAL_PROPERTY;
+  @Input() year: YearInterval = 'cero';
```

Comment:
I think this should be zero :)

Reply:
🤦‍♂️  this makes billy Corgan sad, he wore that shirt for nothing.
Thanks!